app/views/docs/queries.phtml

```diff
@@ -0,0 +1,500 @@
+<p>Many list endpoints in Appwrite allow you to filter, sort, and paginate results using queries. Appwrite provides a common set of syntax to build queries.</p>
+
+<h2><a href="/docs/queries#query-compatibility" id="queryable-compatibility">Query Compatibility</a></h2>
+<p>Each list endpoint supports different query operations. You can find the supported query methods and attributes in the <b>References</b> section of the Appwrite documentation.</p>
+
+<h2><a href="/docs/queries#query-class" id="query-class">Query Class</a></h2>
+<p>Appwrite SDKs provide a <code>Query</code> class to help you build queries. The <code>Query</code> class has a method for each type of supported query.</p>
+<table cellspacing="0" cellpadding="0" border="0" class="full margin-bottom-large">
+    <thead>
+        <tr>
+            <th style="width: 100px">Query Method</th>
+            <th>SDK Method Example</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>Select</td>
+            <td><code>Query.select(["name", "title"])</code></td>
+            <td>Select which attributes should be returned from a document.</td>
+        </tr>
+        <tr>
+            <td>Equal</td>
+            <td><code>Query.equal("title", ["Iron Man"])</code></td>
+            <td>Returns document if attribute is equal to any value in the provided array.</td>
+        </tr>
+        <tr>
+            <td>Not Equal</td>
+            <td><code>Query.notEqual("title", "Iron Man")</code></td>
+            <td>Returns document if attribute is not equal to any value in the provided array.</td>
```

Comment:
```suggestion
            <td><code>Query.notEqual("title", ["Iron Man"])</code></td>
            <td>Returns document if attribute is not equal to any value in the provided array.</td>
```

The example should be an array

Reply:
@christyjacob4 @stnguyen90 Recommended we don't because you should not provide more than 1 value in not equal, because of the OR logic, it will fetch all documents.

A way to discourage is by not showing `[]` in docs. WDYT?

Reply:
Ah okay. I was not aware of that issue. Sounds good then 👍